templates/default/customer/customeradd.html

```diff
@@ -122,17 +122,32 @@ <H1>{$layout.pagetitle}</H1>
 			<div class="lmsbox-panel">
 				<table>
 					<tr>
+						<td>
+							<label for="customeradd[extid]" class="nobr bold">
+								{trans("External system identifier")}
+							</label>
+						</td>
 						<td>
 							<div class="valign-middle">
-								<label>
-									<strong>{trans("External system identifier")}</strong>
-									<input type="text" size="20" maxlength="20" name="customeradd[extid]" value="{$customeradd.extid|escape}"
-										{tip trigger="extid"}>
-								</label>
+								<input type="text" class="lms-ui-autogrow" name="customeradd[extid]" value="{$customeradd.extid|escape}"
+									{tip trigger="extid"}>
 								{hint content="Enter customer identifier in external system (optional)"}
 							</div>
 						</td>
 					</tr>
+					<tr>
+						<td>
+								<label class="bold nobr" for="customeradd[altname]">
```

Comment:
`for` działa na `name` innego elementu? Raczej działa na `id` innego elementu.

Reply:
https://github.com/chilek/lms/pull/2308/commits/4918241c6f8b7f0b269c2e03856aba229950836c

Sugerowałem się tym co było w name i lastname - poprawiłem również i tam.